easybuild/easyconfigs/p/pmt/pmt-1.2.0-GCCcore-11.3.0-CUDA-11.7.0.eb

```
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'pmt'
+version = '1.2.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://git.astron.nl/RD/pmt'
+description = """PMT is a high-level software library capable of 
+    collecting power consumption measurements on various hardware."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+builddependencies = [
+    ('CMake', '3.23.1'),
+    ('binutils', '2.38')
+]
+
+dependencies = [
+    ('CUDA', '11.7.0', '', SYSTEM)
+]
+
+
+source_urls = ['https://git.astron.nl/RD/pmt/-/archive/%(version)s']
+sources = ['pmt-%(version)s.tar.gz']
+checksums = ['98faf305c60cc4d39d3b8ada1dcb7c87c49bf6f01a3cd2800b413d21d5df1a38']
```

Little bit of nitpicking related to style/order:
```suggestion
source_urls = ['https://git.astron.nl/RD/pmt/-/archive/%(version)s']
sources = ['pmt-%(version)s.tar.gz']
checksums = ['98faf305c60cc4d39d3b8ada1dcb7c87c49bf6f01a3cd2800b413d21d5df1a38']

builddependencies = [
    ('CMake', '3.23.1'),
    ('binutils', '2.38'),
]

dependencies = [
    ('CUDA', '11.7.0', '', SYSTEM),
]
```